### PR TITLE
Fixed Git GIt bad bad git

### PR DESCRIPTION
**Pull request by King-Willie**
Ol reliable delete everything and recommit it all your welcome

### CODE REVIEW

**Comment by King-Willie** [comment]
A LOT OF STUFF CLEANED UP

-REMOVED A DUPLICATE PRODUCT CLASS (THERE WAS TWO AND HALF OF EVERY FUNCTION USED DIFFERENT ONES)
-ADDED CHECKOUT FUNCTIONALITY (TAKES IN SAME STRING AS THE CHECKOUT BALANCE BUT RETURNS SUCCESS IF ITEMS AVAlABLE AND THE PHRASE ["Item '" + item.getProductName() + "' is not in stock or does not exist."] OF THE FIRST ITEM THAT IS OUT OF STOCK


Original PR Comment ^^^

File: spring-boot/src/main/java/springboot/testClass.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+//testClass
```
Comment (oliverliu1) [inline spring-boot/src/main/java/springboot/testClass.java:1]: Put Test cases in `src/test/java`
Name them FileNameTest
Use JUnit?

File: spring-boot/src/main/java/springboot/service/CartService.java [hunk 2]
```diff
@@ -29,6 +31,14 @@ public String getPriceCart(String [] items){
         return String.valueOf(price);
     }
 
+    public String shopCheckout(String [] items){
```
Comment (oliverliu1) [inline spring-boot/src/main/java/springboot/service/CartService.java:34]: Inconsistent white space before curly brace

**Review by oliverliu1 — CHANGES_REQUESTED** [review]
Mostly looks good, just move the test file to src/test/main
And name it FileNameTest

**Comment by King-Willie** [comment]
Changes done >:D

**Pull request merged**